Cartfile

```diff
@@ -1,7 +1,7 @@
 binary "https://www.mapbox.com/ios-sdk/MapboxAccounts.json" ~> 2.3.0
-binary "https://mapbox-gl-native-ios.s3.amazonaws.com/public/internal/Mapbox-iOS-SDK.json" == 5.9.1000
 github "mapbox/mapbox-directions-swift" ~> 0.33
-binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" ~> 14.1.5
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps/mapbox-ios-sdk-dynamic.json" ~> 6.0
+binary "https://www.mapbox.com/ios-sdk/MapboxNavigationNative.json" ~> 14.1.6
```

Comment:
Can you rebase #2408 instead of making this change here? I think it’ll end up causing unnecessary merge conflicts later.

Reply:
Sounds good.